From 83ac6c9a151e77dcccfc2582e5a0760f459504e4 Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Mon, 22 Mar 2010 15:47:07 +0000
Subject: [PATCH] Disable font plugin by default

The cursor theme for new apps is already loaded by the XSettings
plugin, and the X core fonts are obsoleted by fontconfig. Handling
of new fonts is done in the xsettings plugin as well.

https://bugzilla.gnome.org/show_bug.cgi?id=613604
---
 data/gnome-settings-daemon.schemas.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/data/gnome-settings-daemon.schemas.in b/data/gnome-settings-daemon.schemas.in
index 16bb71f..139919b 100644
--- a/data/gnome-settings-daemon.schemas.in
+++ b/data/gnome-settings-daemon.schemas.in
@@ -76,7 +76,7 @@
       <applyto>/apps/gnome_settings_daemon/plugins/font/active</applyto>
       <owner>gnome-settings-daemon</owner>
       <type>bool</type>
-      <default>TRUE</default>
+      <default>FALSE</default>
       <locale name="C">
         <short>Enable font plugin</short>
         <long>Set to True to enable the plugin to manage font settings.</long>
-- 
1.7.0.1

